jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/card/events/AnimatedListCell.kt

```diff
@@ -0,0 +1,56 @@
+package org.wycliffeassociates.otter.jvm.controls.card.events
+
+import javafx.animation.TranslateTransition
+import javafx.event.EventHandler
+import javafx.scene.Node
+import javafx.scene.control.ListCell
+import javafx.util.Duration
+
+abstract class AnimatedListCell<T> : ListCell<T>() {
```

Comment:
@AnonymousWalker I would disagree with the proposed names, I prefer AnimatedListCell. This is because AnimatedListCell IS a cell, a cell decorated with animation functionality. 

If Max had made ListItemAnimation or ListCellAnimation, I would expect the classes to solely be an animation, or extend animation, and that the animation would be used as a member of the Cell.

One name (AnimatedListCell) implies an Is-A relationship, the others imply a Has-A relationship